quiz.js

```diff
@@ -10,66 +9,53 @@
 
 // The character to use should be from user input in a <input type="text"> field in the DOM.
 
+// It accepts a single object as an argument. The object should have two key/value pairs.
 
 var tree = {
 	height: userHeight,
 	character: userChar
 };
 
-
-// function growTree(height, char) {
-// 	// body... call variables inside so they don't come up as undefined
-// }
-
-
-
 var treeButton = document.getElementById("yourTree");
 
+//This function is for the Enter button
 
+function enterInput(event) {
+	if (event.keyCode === 13) {
+		checkInput(true);
```

Comment:
Looking back, I'm not sure if I thought I needed to pass a value through to checkInput.